apps/autoscale-by-metric.html.markerb

```diff
@@ -0,0 +1,209 @@
+---
+title: Autoscale based on metrics
+layout: docs
+nav: firecracker
+---
+
+The metrics-based autoscaler scales an app's Machines based on any metric, such as pending work items or queue depth. Scaling on metrics other than requests or connections is useful for apps, like background workers, that aren't running web services. Apps with services that need to scale based on HTTP requests can use the built-in Fly Proxy [autostop/autostart feature](https://fly.io/docs/apps/autostart-stop/) for Machines.
+
+The autoscaler works by collecting metrics from different sources, such as Prometheus or Temporal, and computing the number of required Machines based on those metrics. This reconciliation process happens on a loop every 15 seconds by default. We use the [Expr language](https://expr-lang.org/) for defining target Machine counts, which gives a rich set of built-in arithmetic functions.
```

Comment:
Should we say how you can change the reconciliation interval in the yml config? Can you use a combination of yaml config and env variables?